### PR TITLE
Om 24 change a class to be a container

### PR DESCRIPTION
**Pull request by MartinHartkopf**
Short description Custom Classes

Long definition of custom classes and how to create them

### CODE REVIEW

**Comment by MartinHartkopf** [comment]
Definition of custom Classes

File: docs/admin/custom-classes.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+
+title: create custom classes  
+description: customize i-doit and create custom classes  
+lang: en
+
+---
+
+# custom classes
```
Comment (MichaelOv) [inline docs/admin/custom-classes.md:9]: Headings always begin with uppercase letters

File: docs/admin/custom-classes.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+title: create custom classes  
+description: customize i-doit and create custom classes  
+lang: en
+
+---
+
+# custom classes  
+Classes in i-doit are the basis of your it documentation. i-doit is delivered with a number of predefined classes, which should help you to get started quickly with it documentation.
+
+# add custom classes
```
Comment (MichaelOv) [inline docs/admin/custom-classes.md:12]: Headings always begin with uppercase letters

File: docs/admin/custom-classes.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+But we give you the possibility to define your own classes.
+
+Go to `Settings>CMDB Configuration>Classes` and click on the New Class button.
+
+In the mask that opens, you can now enter a name for the new class and confirm with `Save`.
+After that the new class is available and can be used.
+
+# customizing custom classes
```
Comment (MichaelOv) [inline docs/admin/custom-classes.md:20]: Headings always begin with uppercase letters

**Review by MichaelOv — CHANGES_REQUESTED** [review]

**Review by MichaelOv — APPROVED** [review]

**Pull request merged**